webservices/common/models/audit.py

```diff
@@ -0,0 +1,114 @@
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import TSVECTOR
+from sqlalchemy.ext.declarative import declared_attr
+
+from webservices import docs
+
+from .base import db
+
+class AuditBase(object):
+    __table_args__ = {"schema": "auditsearch"}
```

Comment:
Just stylistic nit-pick for future reference:  we use single quotes (`'`) to wrap strings in Python usually.  No need to change now, just wanted to flag it!